specification/documentation/APIDescription.md

```diff
@@ -161,7 +161,7 @@ We will send your API key in the `x-api-key header`. Your service should respond
 * `401 Unauthorized` if the API key is not received
 * `401 Unauthorized` if the API key is invalid
 
-We will send you a HMAC-SHA256 signature in the `x-hmac-sha256-signature` header. You will need to validate the signature to verify the response has come from an authorized sender. Details on this will be provided during the onboarding process. If you receive a request with an invalid signature you should ignore it and respond with a `403 Forbidden`.
+We will send you a HMAC-SHA256 signature in the `x-hmac-sha256-signature` header. You will need to validate the signature to verify the response has come from an authorized sender. This can be achieved by passing in the body crossed with the applicationID and apikey e.g. `hmacsha256(body, 'applicationId.apiKey')`. If you receive a request with an invalid signature you should ignore it and respond with a `403 Forbidden`.
```

Comment:
I worry `hmacsha256(body, 'applicationId.apiKey')` may be slightly confusing.

Possibly something like `This can be achieved by hashing the request body using the HMAC-SHA256 algorithm with a secret value that is comprised of a concatenation of your APIM application ID and the API key that we provide you. The secret takes the following form <APPLICATION_ID>.<API_KEY>`

Reply:
Atleast make it a suggestion 😆 
```suggestion
This can be achieved by hashing the request body using the HMAC-SHA256 algorithm with a secret value that is comprised of a concatenation of your APIM application ID and the API key that we provide you. The secret takes the following form '<APPLICATION_ID>.<API_KEY>'.
```